dapps/assets/src/components/digitaltwins/AddDigitalTwin.vue

```diff
@@ -28,15 +28,20 @@
       :hideCloseButton="true"
       :title="$t('_assets.digitaltwins.add-digitaltwin-title')"
     >
-      <!-- Use profile picture componnent for now. Might need refactoring to adjust texts. -->
-      <evan-profile-picture
-        type="device"
-        :accountName="name"
-        :isEditable="true"
-        :src="image ? image : null"
-        @changed="handleImageChange"
-      />
-
+      <label class="col-form-label" for="new-twin-picture">
+        {{ '_assets.digitaltwins.add-image' | translate }}
+        <small class="text-muted">({{ '_assets.optional' | translate }})</small>
+      </label>
+      <div class="my-3 centerX">
```

Comment:
You could add `class="d-flex justify-content-center"` to the `evan-profile-picture` element, to avoid new `centerX` class.